src/graphql/mutations/__tests__/UpdateUser.js

```diff
@@ -4,17 +4,49 @@ import client from 'util/client';
 import MockDate from 'mockdate';
 import fixtures from '../__fixtures__/UpdateUser';
 
+const testUser1 = fixtures['/users/doc/testUser1'];
+const testUser2 = fixtures['/users/doc/testUser2'];
+
+const updateUser = (variableString, userId) =>
+  gql`
+      mutation {
+        updatedUser: UpdateUser(${variableString}) {
+          id
+          name
+          slug
+          bio
+          avatarType
+          avatarUrl
+          avatarData
+          updatedAt
+        }
+      }
+    `({}, { userId });
+
+const getUser = async userId => {
+  const {
+    body: { _source: user },
+  } = await client.get({
+    index: 'users',
+    type: 'doc',
+    id: userId,
+  });
+  return user;
+};
+
 describe('UpdateUser', () => {
+  let now = 1485593157011;
+  beforeAll(() => loadFixtures(fixtures));
   beforeEach(() => {
-    MockDate.set(1485593157011);
-    return loadFixtures(fixtures);
+    MockDate.set(now);
+    now += 10000;
```

Comment:
Does this mean that each test case will receive different date?
If we add new test case between existing test cases, does all subsequent test cases' `updatedAt` also change?